packages/core-svelte/src/lib/components/TextComponent.svelte

```diff
@@ -688,9 +687,9 @@
     <ErrorMarker {editor} {box} />
 {/if}
 <ErrorTooltip {editor} {box} {hasErr} parentTop={0} parentLeft={0}>
-    <span {id} role="none" bind:this={surroundingElement}>
+    <span {id} role="none" bind:this={surroundingElement} class="text-component-parent">
         {#if isEditing}
-            <span class="text-component-input">
+            <span class="text-component-input-wrapper">
```

Comment:
No problem making this change, but why would you need separate CSS classes on the \<span> and the \<input>?